### ISSUE TITLE
ansible-bender build hangs

### ISSUE THREAD

**Issue by jpopelka** [ansible-community/ansible-bender#101]
```
$ git clone https://github.com/packit-service/packit.git
$ ansible-bender build -- ./recipe.yaml
```

It hangs indefinitely. With `--debug` I can see
```
utils.py DEBUG  TASK [copy]  ****
utils.py DEBUG  fatal: [localhost]: FAILED! => {"changed": false, "checksum": "71dfb412ae517d2a4442e26331f29f5d9ecc7dd1", "msg": "Aborting, target uses selinux but python bindings (libselinux-python) aren't installed!"}
```

**Comment by dhodovsk** [comment]
I got the same hang issue but with a different error. 
```
10:01:44.704 utils.py          DEBUG  fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'home_path' is undefined...
```

**Comment by TomasTomecek** [comment]
@jpopelka this feels like a bug in ansible, I can check for such case in bender so this won't happen

@dhodovsk that should be fixed in 0.5.1, please create a new issue if you are still hitting it

**Comment by TomasTomecek** [comment]
Reproduced locally.

https://bugzilla.redhat.com/show_bug.cgi?id=1696706
